### PR TITLE
VR-6394 In repo notebooks, replace pool.map()s with for loops

### PR DESCRIPTION
**Pull request by nhatsmrt**
Is there a way to hide outputs 🤔

### CODE REVIEW

File: client/workflows/examples/xgboost.ipynb [hunk 1]
```diff
@@ -192,7 +358,26 @@
...
+      "{'eta': 0.5, 'max_depth': 3, 'num_class': 10} Mean error: 0.0394\n",
+      "created new ExperimentRun: Run 6508415990677575566201\n",
+      "{'eta': 0.7, 'max_depth': 1, 'num_class': 10} Mean error: 0.0503\n",
+      "created new ExperimentRun: Run 650841599067803123082\n",
+      "{'eta': 0.7, 'max_depth': 2, 'num_class': 10} Mean error: 0.0340\n",
+      "created new ExperimentRun: Run 650841599067847944371\n",
+      "{'eta': 0.7, 'max_depth': 3, 'num_class': 10} Mean error: 0.0394\n"
+     ]
+    }
+   ],
```
Comment (convoliution) [inline client/workflows/examples/xgboost.ipynb]:
```suggestion
   "outputs": [],
```

File: client/workflows/examples/xgboost.ipynb [hunk 2]
```diff
@@ -171,7 +325,19 @@
...
+     "output_type": "stream",
+     "text": [
+      "set email from environment\n",
+      "set developer key from environment\n",
+      "connection successfully established\n",
+      "got existing Project: Wine Multiclassification\n",
+      "got existing Experiment: Boosted Trees\n"
+     ]
+    }
+   ],
```
Comment (convoliution) [inline client/workflows/examples/xgboost.ipynb]:
```suggestion
   "outputs": [],
```

File: client/workflows/examples/xgboost.ipynb [hunk 3]
```diff
@@ -132,7 +131,162 @@
...
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
```
Comment (convoliution) [inline client/workflows/examples/xgboost.ipynb]: My vertical monitor isn't tall enough to highlight this full diff 😆 
```JSON
   "outputs": [],
```

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb [hunk 4]
```diff
@@ -187,7 +452,15 @@
...
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Training accuracy: 0.8432\n"
+     ]
+    }
+   ],
```
Comment (convoliution) [inline client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]:
```suggestion
   "outputs": [],
```

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb [hunk 5]
```diff
@@ -169,7 +423,18 @@
...
+     "data": {
+      "text/plain": [
+       "LogisticRegression()"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```
Comment (convoliution) [inline client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]:
```suggestion
   "outputs": [],
```

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb [hunk 6]
```diff
@@ -135,7 +362,34 @@
...
+      "{'C': 10, 'solver': 'liblinear', 'max_iter': 28} Validation accuracy: 0.8473\n",
+      "{'C': 10, 'solver': 'lbfgs', 'max_iter': 15} Validation accuracy: 0.7928\n",
+      "{'C': 10, 'solver': 'lbfgs', 'max_iter': 28} Validation accuracy: 0.7930\n",
+      "{'C': 1000.0, 'solver': 'liblinear', 'max_iter': 15} Validation accuracy: 0.7939\n",
+      "{'C': 1000.0, 'solver': 'liblinear', 'max_iter': 28} Validation accuracy: 0.8473\n",
+      "{'C': 1000.0, 'solver': 'lbfgs', 'max_iter': 15} Validation accuracy: 0.7928\n",
+      "{'C': 1000.0, 'solver': 'lbfgs', 'max_iter': 28} Validation accuracy: 0.7930\n"
+     ]
+    }
+   ],
```
Comment (convoliution) [inline client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]:
```suggestion
   "outputs": [],
```

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb [hunk 7]
```diff
@@ -86,7 +85,235 @@
...
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
```
Comment (convoliution) [inline client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]: See suggestions below
```json
   "outputs": [],
```

File: client/workflows/demos/census-end-to-end.ipynb [hunk 8]
```diff
@@ -227,7 +466,58 @@
...
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "created new ExperimentRun: Run 656741599067982385035\n",
+      "{'C': 1e-06, 'solver': 'lbfgs', 'max_iter': 15} Validation accuracy: 0.7902\n",
+      "uploading part 1\n",
+      "upload complete (custom_modules)\n",
+      "uploading part 1\n",
```
Comment (convoliution) [inline client/workflows/demos/census-end-to-end.ipynb]: See suggestions below
```json
   "outputs": [],
```

File: client/workflows/demos/census-end-to-end.ipynb [hunk 9]
```diff
@@ -204,7 +431,19 @@
...
+     "output_type": "stream",
+     "text": [
+      "set email from environment\n",
+      "set developer key from environment\n",
+      "connection successfully established\n",
+      "got existing Project: Census Income Classification\n",
+      "got existing Experiment: Logistic Regression\n"
+     ]
+    }
+   ],
```
Comment (convoliution) [inline client/workflows/demos/census-end-to-end.ipynb]:
```suggestion
   "outputs": [],
```

File: client/workflows/demos/census-end-to-end.ipynb [hunk 10]
```diff
@@ -159,7 +158,235 @@
...
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
```
Comment (convoliution) [inline client/workflows/demos/census-end-to-end.ipynb]: See suggestions below
```json
   "outputs": [],
```

**Review by convoliution — CHANGES_REQUESTED** [review]
There's a way https://github.com/VertaAI/modeldb/blob/master/client/CONTRIBUTING.md#example-and-demo-writing to hide cell outputs when saving a notebook, but I wouldn't want to cause you to lose any actual datasci work you do 😆

File: client/workflows/examples/xgboost.ipynb (hunk 1 above)
Comment (nhatsmrt) [reply client/workflows/examples/xgboost.ipynb]: Fixed.

File: client/workflows/examples/xgboost.ipynb (hunk 2 above)
Comment (nhatsmrt) [reply client/workflows/examples/xgboost.ipynb]: Fixed.

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb (hunk 4 above)
Comment (nhatsmrt) [reply client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]: Fixed.

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb (hunk 5 above)
Comment (nhatsmrt) [reply client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]: Fixed.

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb (hunk 6 above)
Comment (nhatsmrt) [reply client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]: Fixed.

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb (hunk 7 above)
Comment (nhatsmrt) [reply client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]: Fixed.

File: client/workflows/demos/census-end-to-end.ipynb (hunk 8 above)
Comment (nhatsmrt) [reply client/workflows/demos/census-end-to-end.ipynb]: Fixed.

File: client/workflows/demos/census-end-to-end.ipynb (hunk 9 above)
Comment (nhatsmrt) [reply client/workflows/demos/census-end-to-end.ipynb]: Fixed.

File: client/workflows/demos/census-end-to-end.ipynb (hunk 10 above)
Comment (nhatsmrt) [reply client/workflows/demos/census-end-to-end.ipynb]: Fixed.

**Comment by nhatsmrt** [comment]
@convoliution all outputs removed.

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb [hunk 11]
```diff
@@ -154,8 +153,8 @@
     "    print(hyperparams, end=' ')\n",
     "    print(\"Validation accuracy: {:.4f}\".format(val_acc))\n",
     "    \n",
-    "with Pool() as pool:\n",
-    "    pool.map(run_experiment, hyperparam_sets)"
+    "for hyperams in hyperparam_sets:\n",
+    "    run_experiment(hyperams)"
```
Comment (convoliution) [inline client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]:
```suggestion
    "# NOTE: run_experiment() could also be defined in a module, and executed in parallel\n",
    "for hyperparams in hyperparam_sets:\n",
    "    run_experiment(hyperparams)"
```

File: client/workflows/examples/xgboost.ipynb [hunk 12]
```diff
@@ -215,8 +214,9 @@
     "    run.log_metric(\"val_error\", final_val_error)\n",
     "    print(\"{} Mean error: {:.4f}\".format(hyperparams, final_val_error))\n",
     "    \n",
-    "with multiprocessing.Pool() as pool:\n",
-    "    pool.map(run_experiment, grid)"
+    "# NOTE: run_experiment() could also be defined in a module, and executed in parallel\n",
+    "for hyperams in grid:\n",
+    "    run_experiment(hyperams)"
```
Comment (convoliution) [inline client/workflows/examples/xgboost.ipynb]:
```suggestion
    "for hyperparams in grid:\n",
    "    run_experiment(hyperparams)"
```

File: client/workflows/demos/census-end-to-end.ipynb [hunk 13]
```diff
@@ -263,9 +262,9 @@
     "    # log Git information as code version\n",
     "    run.log_code()\n",
     "    \n",
-    "pool = multiprocessing.Pool()\n",
-    "pool.map(run_experiment, hyperparam_sets)\n",
-    "pool.close()"
+    "# NOTE: run_experiment() could also be defined in a module, and executed in parallel\n",
+    "for hyperams in hyperparam_sets:\n",
+    "    run_experiment(hyperams)"
```
Comment (convoliution) [inline client/workflows/demos/census-end-to-end.ipynb]: Bit of a tonguetwister: it's **hyper**-**param**eters
```suggestion
    "for hyperparams in hyperparam_sets:\n",
    "    run_experiment(hyperparams)"
```

**Review by convoliution — CHANGES_REQUESTED** [review]

File: client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb (hunk 11 above)
Comment (nhatsmrt) [reply client/workflows/examples-without-verta/notebooks/sklearn-census.ipynb]: Fixed.

File: client/workflows/examples/xgboost.ipynb (hunk 12 above)
Comment (nhatsmrt) [reply client/workflows/examples/xgboost.ipynb]: Fixed.

File: client/workflows/demos/census-end-to-end.ipynb (hunk 13 above)
Comment (nhatsmrt) [reply client/workflows/demos/census-end-to-end.ipynb]: Fixed.

**Review by conradoverta — APPROVED** [review]

**Review by convoliution — APPROVED** [review]

**Pull request merged**